frontend/src/metabase/visualizations/echarts/cartesian/waterfall/model/dataset.ts

```diff
@@ -41,23 +20,24 @@ function createDatum({
   total,
 }: {
   dimension: string;
-  barOffset?: number | null;
-  increase?: number | null;
-  decrease?: number | null;
+  barOffset?: number;
+  increase?: number | WaterfallEmptyValue;
+  decrease?: number | WaterfallEmptyValue;
   total?: number;
 }): WaterfallDatum {
   return {
     [DATASET_DIMENSIONS.dimension]: dimension,
     [DATASET_DIMENSIONS.barOffset]: barOffset ?? 0,
-    [DATASET_DIMENSIONS.increase]: increase ?? null,
-    [DATASET_DIMENSIONS.decrease]: decrease ?? null,
-    [DATASET_DIMENSIONS.total]: total ?? null,
+    [DATASET_DIMENSIONS.increase]: increase ?? "-",
```

Comment:
Using a constant instead of "-" could make it more readable

Reply:
Fixed in https://github.com/metabase/metabase/pull/37277